runtime/src/mocks.rs

```diff
@@ -1,15 +1,19 @@
 //! A mock implementation of a channel that implements the Sink and Stream traits.
 
-use crate::{Error, Sink, Stream};
+use crate::{
+    Error,
+    Sink as SinkTrait,
+    Stream as StreamTrait,
+};
 use bytes::Bytes;
 use futures::channel::oneshot;
 use std::{
     collections::VecDeque,
     sync::{Arc, Mutex},
 };
 
-// A mock channel struct that is used internally by ByteSink and ByteStream.
-struct ByteChannel {
+// A mock channel struct that is used internally by Sink and Stream.
```

Comment:
If you want any of these comments to appear in docs, they need to start with `///`.

You can test out how it would look by running `cargo doc --open` inside of a crate (if you see any links without some documentation, it probably means you used `//` by accident).

Reply:
(I'd suggest you do at least for the public structs/functions in the mock crate).

`//` is reserved for just code comments you don't need to appear publicly for developers.